dom/ajax/ajax.js

```diff
@@ -73,9 +73,11 @@ $._xhrResp = function (xhr, options) {
 	}
 };
 $._formData = function (o) {
-	var kvps = [], regEx = /%20/g;
+	var kvps = [], regEx = /%20/g, val;
 	for (var k in o) {
-		kvps.push(encodeURIComponent(k).replace(regEx, "+") + "=" + encodeURIComponent(o[k].toString()).replace(regEx, "+"));
+		val = o[k];
+		val = typeof val === "undefined" || val === null ? "" : val;
```

Comment:
If eshint allows it, I usually write `val == null`.  Comparing against `null` or `undefined` is really the only good place for `==`.  

Also, you could just do `val === undefined` instead of using typeof.